apps/courses/src/routes/index.tsx

```diff
@@ -3,11 +3,13 @@ import { Navigate, Route, Routes } from 'react-router-dom';
 import { useUser } from 'reactfire';
 
 const Homepage = lazy(() => import('./homepage'));
-const Signup = lazy(() => import('./sign-up'));
-const Signin = lazy(() => import('./sign-in'));
-const AccountSettings = lazy(() => import('./account-settings'));
-const Profile = lazy(() => import('./profile'));
+const Signup = lazy(() => import('./users/sign-up'));
+const Signin = lazy(() => import('./users/sign-in'));
+const Settings = lazy(() => import('./users/settings'));
+const Profile = lazy(() => import('./users/profile'));
+const CourseOverview = lazy(() => import('./courses/overview'));
 const NoMatch = lazy(() => import('./no-match'));
+const Courses = lazy(() => import('./courses'));
```

Comment:
First of all, move this above `NoMatch` so that we keep the EXACT same order in the export below.

Reply:
And second of all, move this to a folder under `/courses/search`

Reply:
Also rename the component to be called `CoursesSearch`.